org/mozilla/jss/ssl/javax/JSSEngine.java

```diff
@@ -945,7 +945,6 @@ public boolean isOutboundDone() {
         // TLS < 1.3.
         result.put(SSL.ENABLE_RENEGOTIATION, SSL.RENEGOTIATE_REQUIRES_XTN);
         result.put(SSL.REQUIRE_SAFE_NEGOTIATION, 1);
-        result.put(SSL.ENABLE_FALLBACK_SCSV, 1);
```

Comment:
would it makes sense to just comment this out and give explanation, so later someone who wishes to add this would know why?

Reply:
@cipherboy I"m not sure if I've seen any response to this ;-)

Reply:
Ah sorry, I responded at the bottom: 

https://github.com/dogtagpki/jss/pull/627#issuecomment-685190217

The version in `master` has a better commit message and description: 

https://github.com/dogtagpki/jss/commit/d0c8b2453dff73d412ba7a4ffa274c16ea57a415

```java

        // Don't enable FALLBACK_SCSV support; this is incompatible with other
        // JCA providers and makes transitioning TLS versions difficult.
        // result.put(SSL.ENABLE_FALLBACK_SCSV, 1);

```

Reply:
Ah, I didn't know you were responding to this one, as I was suggesting you to comment out that line instead of removing it in the code and add comment in the code.

Reply:
it's not that critical though. It's just a suggestion.

Reply:
Yeah it was commented out (as suggested here) in `master` for posterity. :-)

Reply:
Ah, sorry, my bad.  I didn't see. ;-) Thanks!!